src/app/components/source/SourceComponent.js

```diff
@@ -1054,7 +1053,7 @@ class SourceComponent extends Component {
 
           { !isEditing ?
             <div>
-              { this.state.showTab === 'annotation' ? <Annotations annotations={source.annotations.edges.slice().reverse()} annotated={source} annotatedType="Source" /> : null }
+              { this.state.showTab === 'annotation' ? <Annotations annotations={source.log.edges} annotated={this.props.source} annotatedType="ProjectSource" /> : null }
```

Comment:
This might be confusing because the data is structured like this: `this.props.source.source`, where the parent is the `project_source` and the child is the `source`. I don't like this naming but I followed how it was like for `media`.